libdnf/base/base.cpp

```diff
@@ -105,6 +105,7 @@ void Base::setup() {
     auto & config = get_config();
     auto & installroot = config.installroot();
     get_vars()->load(installroot.get_value(), config.varsdir().get_value());
+    system_state.emplace(installroot.get_value());
```

Comment:
What about to modify `auto & installroot = config.installroot();` by `auto & installroot = config.installroot().get_value();`.

It is not important.

Reply:
Well then I couldn't use the variable for calling `lock()` on the line directly below this one so I think it comes out quite equal...

Reply:
Ok, good point